corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -97,6 +103,8 @@ hqDefine("data_dictionary/js/data_dictionary", [
                     dataType: 'JSON',
                     data: {
                         'properties': JSON.stringify(postProperties),
+                        'fhir_resource_type': self.fhirResourceType(),
+                        'case_type': self.activeCaseType(),
```

Comment:
Here we are creating the JSON to send for saving various case properties. Each case prop object also has the same "case_type" name added and then when processing the request, we iterate on each case prop and process it.
Now I need the case_type on backend before iterating the props passed to create fhir resource type. I could have read from one of the case properties passed but its not necessary that there will be any, unlikely but possible. So I have passed it separately in the request.